stable/database/README.md

```diff
@@ -310,6 +310,25 @@ The purpose of this section is to allow customisation of the names of the cluste
 | `clusterip` | suffix for the clusterIP load-balancer | .Values.admin.serviceSuffix.clusterip |
 | `balancer` | suffix for the balancer service | .Values.admin.serviceSuffix.balancer |
 
+#### nuocollector.*
+
+The purpose of this section is to specify the NuoDB monitoring parameters.
+
+The following tables list the configurable parameters for the `nuocollector` option of the admin chart and their default values.
+
+| Parameter | Description | Default |
+| ----- | ----------- | ------ |
+| `enabled` | Whether to enable NuoDB monitoring using sidecar containers |`false`|
+| `image.registry` | NuoDB Collector container registry | `docker.io` |
+| `image.repository` | NuoDB Collector container image name |`nuodb/nuodb-collector`|
+| `image.tag` | NuoDB Collector container image tag | `1.0.1` |
```

Comment:
this value is incorrect. The chart uses 1.1.0

Reply:
Good catch! I forgot to change this one.